src/main/java/command/FindCommand.java

```diff
@@ -0,0 +1,35 @@
+package command;
+import entry.Task;
+import oracle.TaskList;
+import oracle.Ui;
+
+import java.util.ArrayList;
+
+public class FindCommand implements Command{
+
+    private final String keyword;
+
+    /**
+     * @param keyword String of the keyword to be searched
+     */
+    public FindCommand(String keyword) {
+        this.keyword = keyword;
+    }
+
+    /**
+     * @param ui we give the user the results of the search
+     * @param tasks we find any relevant tasks from this TaskList
+     * @return true
+     */
+    @Override
+    public boolean execute(Ui ui, TaskList tasks) {
+        ArrayList<Task> results = new ArrayList<>();
+        for (Task t : tasks.getTasks()) {
+            if (t.toString().contains(this.keyword)) {
```

Comment:
I think can give a variable to t.toString() to explain whether this is the task description or the string representation of the task.